app/models/rdv.rb

```diff
@@ -2,7 +2,6 @@
 
 class Rdv < ApplicationRecord
   include WebhookDeliverable
-  include Rdv::NotifiableConcern
```

Comment:
C’est la plus grosse différence: je crois que j’ai repéré tous les endroits où il fallait appeler les _notificationservice_ ou _RdvUpdater_ à la main, mais ce n’est pas évident à tester. Je veux bien qu’on fasse un tour des scenarios ensemble @yaf

Reply:
ok, c'est noté.